deep_quoridor/src/agents/alphazero.py

```diff
@@ -1,12 +1,19 @@
+import math
```

Comment:
This file is still WIP, right? Could you add a high-level comment clarifying that? Ideally a quick sentence on what is done and what is missing, which can be updated / removed when this work is completed.

Alternatively, why don't you omit these changes on this PR, and only send the MCTS part here? That may be better, so that the PR shows more clearly only what was needed to implement MCTS